test/formatGeneratedModule-test.ts

```diff
@@ -75,4 +75,18 @@ describe('formatGeneratedModule', () => {
       sourceHash: 'edcba',
     })).toMatchFile(join(__dirname, 'fixtures/generated-module/complete-example-no-cast.ts'))
   })
+
+  it('works with devOnlyAssignments', () => {
+    const formatGeneratedModule = formatterFactory();
+    expect(formatGeneratedModule({
+      moduleName: 'complete-example',
+      documentType: 'ConcreteFragment',
+      docText: null,
+      concreteText: JSON.stringify({ the: { fragment: { data: 42 } }}),
+      typeText: 'export type CompleteExample = { readonly id: string }',
+      hash: 'abcde',
+      sourceHash: 'edcba',
+      devOnlyAssignments: '(node/*: any*/).params.text = "query CompleteExampleQuery { id }";',
```

Comment:
As per the upstream behaviour, the actual query text should be taken from the generated AST node: https://github.com/facebook/relay/blob/42033ebcc298af7e8ed90632b70077b7545a282d/packages/relay-compiler/codegen/writeRelayGeneratedFile.js#L110

Reply:
Scratch this, I wasn’t paying attention to the filename, as per my below comment.